runtime/Java/src/org/antlr/v4/runtime/misc/FlexibleHashMap.java

```diff
@@ -64,9 +64,9 @@ public FlexibleHashMap(AbstractEqualityComparator<? super K> comparator, int ini
 		this.initialBucketCapacity = initialBucketCapacity;
 	}
 
-	private static <K, V> LinkedList<Entry<K, V>>[] createEntryListArray(int length) {
+	private static <K, V> LinkedList<Entry<K, V>>[] createEntryListArray(int capacity) {
```

Comment:
💭 `length` appears to be the more correct term here. Input particular, note that the result has a `length` field which corresponds to this parameter, and does not have a `capacity` field at all.